### PR TITLE
[WIP] Render a jinja2 template for the kubevirt manifest.

### PR DESCRIPTION
**Pull request by dcritch**
This change allows kubevirt-ansible to consume a jinja2 template rather than a pre-rendered manifest from kubevirt.

For now, we include a jinja2 version, but the aim is to have upstream kubevirt maintain it. Otherwise we'll have to set up some system to automatically create a new j2 when upstream changes.

### CODE REVIEW

**Comment by dcritch** [comment]
ci test please

**Comment by fabiand** [comment]
Is there actually already a PR on kubevirt/kubevirt to create the template?

**Comment by dcritch** [comment]
@fabiand No, just an issue. This PR in kubevirt-ansible just gets this project ready to consume it.

**Comment by fabiand** [comment]
@dcritch alright - could you take care of the kubevirt/kubevirt side as well?

File: roles/kubevirt/tasks/provision.yml [hunk 1]
```diff
@@ -25,41 +25,21 @@
   command: "oc adm policy add-scc-to-user hostmount-anyuid -z kubevirt-infra -n {{ namespace }}"
   when: cluster=="openshift"
 
-- name: Check for kubevirt.yaml template in {{ kubevirt_template_dir }}
+- name: Check for kubevirt.yaml.j2 template in {{ kubevirt_template_dir }}
   stat:
-    path: "{{ kubevirt_template_dir }}/kubevirt.yaml"
+    path: "{{ kubevirt_template_dir }}/kubevirt.yaml.j2"
   register: byo_template
```
Comment (rthallisey) [inline roles/kubevirt/tasks/provision.yml]: I think we need to keep these tasks so offline works for upstream images. They will need to change to work with the .j2 extension.
```-- name: Check for offline v{{ version }} templates in {{ offline_template_dir }}```
```-- name: Copy offline templates to /tmp```

Comment (dcritch) [reply roles/kubevirt/tasks/provision.yml]: Adding a couple commits to address these. Thanks!

**Comment by dcritch** [comment]
This thing got messy w/ all the rebases. Closing. Please see [new PR 279](https://github.com/kubevirt/kubevirt-ansible/pull/279)

**Pull request closed without merging**